numpy/lib/stride_tricks.py

```diff
@@ -197,6 +197,43 @@ def _broadcast_shape(*args):
     return b.shape
 
 
+@set_module('numpy')
+def broadcast_shapes(*args):
+    """
+    Get Broadcast shape from the given shape tuples.
```

Comment:
Perhaps to match the function name
```suggestion
    Broadcast the input shapes into a single shape.
```
Also, it would be good to add a reference to the numpy docs on broadcasting here, via ``:ref:`basics.broadcasting```